### DISCUSSION TITLE
contributing/development/configuring_an_ide/rider

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#294] [User-contributed notes]
# contributing/development/configuring_an_ide/rider

Rider is a commercial JetBrains IDE for C# and C++ that uses the same solution system as Visual Studio. Importing the project: Rider requires a solution file to work on a project. While Godot does ...

https://docs.godotengine.org/en/stable/contributing/development/configuring_an_ide/rider.html

**Comment by Justin-Archmagus** [comment]
Here is the JetBrains Rider page on Godot: https://www.jetbrains.com/help/rider/Godot.html with more information.

At the time of this writing, the JetBrains Godot page explains that the Godot plugin is built into Rider by default, but users may want to download the GDScript plugin for improved GDScript support (https://plugins.jetbrains.com/plugin/20123-gdscript).

**Reply by jking47** [reply]
Just to keep this comment thread updated for the docs, the external GDScript plugin mentioned is not supported by the dev anymore at this point (May 30 2025). They say that the Jetbrains team will be taking it over, and as far as I can tell the external one doesn't add much, so installing that is probably unnecessary.
